### DISCUSSION TITLE
LQAS shows Fingermark as Yes and Missed reason also

### DISCUSSION THREAD

**Discussion by naveeddil** [xlg8/APMIS-Project#655] [Q&A]
[
![LQAS issue](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/1c9ad345-cc35-4259-b6da-e1ae2ae18450)
](url)

You can get multiple entries where child is Fingermarked but still there are Missed reason.
As a rule if the child is FM then the missed reasons should not be shown

**Comment by sundayAdu** [comment]
I think this must be form rule based issue, @RaziaMahmodi can provide more clarity on this issue so that we can properly categorise it as a bug or Form Json logic error.

**Comment by RaziaMahmodi** [comment]
@sundayAdu FYI.
@naveeddil this issue is fixed, will be available on live server in new update.
